### PR TITLE
Add vip sandbox run

### PR DESCRIPTION
**Pull request by joshbetz**
See #77 

```
$ vip sandbox run 59 -- wp user list --field=ID 
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
Running command on container:
-- Command: wp user list --field=ID
-- Container: v2joshbetzcom_web_dev_0031
-- Site: v2.joshbetz.com (#59)
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
Are you sure? y
3
2
4
```

### CODE REVIEW

**Comment by joshbetz** [comment]
cc @philipjohn

**Comment by mjangda** [comment]
Thoughts on adding a confirmation step? (And maybe a flag to bypass the confirmation when doing operations in bulk.)

File: src/bin/vip-sandbox.js [hunk 1]
```diff
@@ -41,6 +41,44 @@ program
...
+		utils.findSite( site, ( err, site ) => {
+			if ( err ) {
+				return console.error( err );
+			}
+
+			if ( ! site ) {
+				return console.error( 'Specified site does not exist. Try the ID.' );
+			}
+
+			sandbox.getSandboxForSite( site, ( err, sbox ) =>  {
```
Comment (mjangda) [inline src/bin/vip-sandbox.js]: Feels like this could be abstracted into a new method (as it's largely the same as what happens for sandbox start.

Comment (joshbetz) [inline src/bin/vip-sandbox.js]: Yep, I was thinking the same thing.

**Comment by joshbetz** [comment]
Passing options to the internal command can be kind of confusing.

TL;DR: It's easiest to always use a double dash before the command you want to run.

These all work:

```
$ vip sandbox run 59 -- wp user list
```
```
$ vip sandbox run 59 -- wp user list
```
```
$ vip sandbox run 59 -- wp user list --field=ID
```
```
$ vip sandbox run 59 wp user list
```

This does not work:

```
$ vip sandbox run 59 wp user list --field=ID
```

To understand why, we have to look at what happens when you run `vip` subcommands.

1. vip.js runs and parses the command
2. The args are "normalized", splitting `--foo=bar` to `[--foo, bar]` https://github.com/tj/commander.js/blob/master/index.js#L455
3. `executeSubCommand` is called with the new, normalized args

This is fine for Commander.js commands because it expects options to have spaces between the key and value. WP-CLI requires the `=` though. Since we don't know the nature of WP-CLI options (are they boolean options? Should we expect the next argument to be the value to the previous?) it's hard to parse the new, parsed args and put them back together.

So, we can short-circuit the normalization with a double dash `--`. https://github.com/tj/commander.js/blob/master/index.js#L590 This effectively skips the normalization step and the arguments are passed to the subcommand without being split.

**Comment by joshbetz** [comment]
https://github.com/tj/commander.js/pull/612 would fix the above issue.
